libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts

```diff
@@ -1207,4 +1207,35 @@ describe('DropdownComponent', () => {
       });
     });
   });
+
+  describe('when inside form', () => {
+    let spectator: SpectatorHost<DropdownComponent>;
+    let buttonElement: HTMLButtonElement;
+
+    const createHost = createHostFactory({
+      component: DropdownComponent,
+      declarations: [
+        MockComponents(
+          ButtonComponent,
+          CardComponent,
+          IconComponent,
+          ItemComponent,
+          PopoverComponent
+        ),
+      ],
+    });
+
+    beforeEach(() => {
+      spectator = createHost(`<kirby-dropdown></kirby-dropdown>`, {
+        props: {
+          items: items,
+        },
+      });
+      buttonElement = spectator.queryHost('button[kirby-button]');
+    });
+
+    it('should have type="button" attribute', () => {
+      expect(buttonElement).toHaveAttribute('type', 'button');
+    });
+  });
 });
```

Comment:
The test isn't testing the kirby dropdown element inside a form. 
Consider removing everything except the `it('should have type...` clause and its contents. I think the `it` clause can be moved inside the `by default` describe block around line 7.

Reply:
I was thinking almost the same. I'll move it to `by default`